### PR TITLE
Get sheet data

### PR DESCRIPTION
**Pull request by owenpigott**
Implemented connection to Google Drive. This will output the count of files found today.

### CODE REVIEW

File: lib_src/lib/local_main.py [hunk 1]
```diff
@@ -1,4 +1,8 @@
-from lib_src.lib.main import spl_lambda_handler
+from lib_src.lib.main import self_isolation_lambda_handler
+from dotenv import load_dotenv
+
 
 if __name__ == '__main__':
-    spl_lambda_handler('event', 'context')
+    load_dotenv()
```
Comment (kosiakkatrina) [inline lib_src/lib/local_main.py:6]: I don't think we need this and line 2

File: lib_src/lib/main.py [hunk 2]
```diff
@@ -139,6 +141,11 @@ def self_isolation_lambda_handler(event, context):
         key_file_location
     )
 
+    self_isolation_inbound_folder_id = getenv("SELF_ISOLATING_INBOUND_FOLDER_ID")
+    self_isolation_outbound_folder_id = getenv("SELF_ISOLATING_OUTBOUND_FOLDER_ID")
```
Comment (kosiakkatrina) [inline lib_src/lib/main.py:145]: Do we need to add these env vars through the serverless +AWS param store?

File: lib_src/lib/local_main.py (hunk 1 above)
Comment (Duslerke) [inline lib_src/lib/local_main.py:6]: I don't remember if we needed this in the end? Was env vars not being loaded part of the problem?

**Review by Duslerke — APPROVED** [review]

File: lib_src/lib/main.py (hunk 2 above)
Comment (Duslerke) [reply lib_src/lib/main.py:145]: Didn't we do it on previous PR when we were changing a Terraform file?

Comment (Duslerke) [reply lib_src/lib/main.py:145]: Oh, nevermind. I forgot we skipped them:
https://github.com/LBHackney-IT/here-to-help-data-ingestion/pull/27/files#diff-5777c177609d78dbc2d38ece628e3e9f6c4d602be4723966a0fd0781b9e39057R225-R230

File: lib_src/lib/local_main.py (hunk 1 above)
Comment (owenpigott) [reply lib_src/lib/local_main.py]: Removed 👍

File: lib_src/lib/main.py (hunk 2 above)
Comment (owenpigott) [reply lib_src/lib/main.py]: Added. I also renamed back to SELF_ISOLATION and ensured this is consistent throughout.

**Pull request merged**